### ISSUE TITLE
Fixing the typos

### ISSUE THREAD

**Issue by NIKHIL0VERMA** [AryaChakraborty/articulus_rag#32]
Hey @AryaChakraborty,
I found several typos. 
Fox ex:- src/llms/azurechat.py line 62 `AI assitant` should be `AI assistant`

Assign this to me, I would like to fix them. Kindly add the GSSOC'24 label and level.

**Comment by NIKHIL0VERMA** [comment]
Hey @AryaChakraborty 
I pointed the typo so I should be assigned to solve it as per GSSOC guidelines.
There are other typos too, fixing is not a big task but finding them is. 

What's the profit of raising an issue under GSSOC if PA solve itself. We're not getting any point.

**Comment by NIKHIL0VERMA** [comment]
Hey @AryaChakraborty,
Thankx for adding labels, kindly assigned this to me so that I can fix this ASAP.

**Comment by NIKHIL0VERMA** [comment]
Hey @AryaChakraborty,
this is not the complete fix. There are more typos left

**Comment by NIKHIL0VERMA** [comment]
Guidelines is to work only when an issue is assigned. Should I send PR without getting assigned???

Also be communicative.

**Comment by AryaChakraborty** [comment]
@NIKHIL0VERMA I think that is self explanatory

**Comment by ananyabansal16** [comment]
If this issue is still pending, I can take this up under GSSoC'24 and fix this ASAP. Lemme know. :)

**Comment by venkat1924** [comment]
I would love to continue this!! I plan to use ChatGPT via an API key to automate the task and scan all the content in this repo.

**Comment by AryaChakraborty** [comment]
@venkat1924 please try to use open source resources, since I believe OpenAI's API is paid and closed source

**Comment by venkat1924** [comment]
> @venkat1924 please try to use open source resources, since I believe OpenAI's API is paid and closed source

Agreed, I will definitely look into that. In the mean time, I can use Grammarly and manually correct documents until I prepare the script, as I just did with the contribution guidelines.

**Comment by venkat1924** [comment]
> > @venkat1924 please try to use open source resources, since I believe OpenAI's API is paid and closed source
> 
> Agreed, I will definitely look into that. In the mean time, I can use Grammarly and manually correct documents until I prepare the script, as I just did with the contribution guidelines.

@AryaChakraborty Could you please assign this issue to me!
